addons/mass_mailing/models/mass_mailing.py

```diff
@@ -620,6 +621,15 @@ def onchange_mass_mailing_campaign_id(self, cr, uid, id, mass_mailing_campaign_i
                    'medium_id': mass_mailing_campaign[0].medium_id.id}
             return {'value': dic}
 
+    def create(self, cr, uid, vals, context=None):
+        if not vals['mass_mailing_campaign_id']:
```

Comment:
Well in that case, you're criticizing the initial specification, which has been validated by Antony.